website/docs/d/cloud_backup_schedule.html.markdown

```diff
@@ -101,4 +101,12 @@ In addition to all arguments above, the following attributes are exported:
 * `retention_unit` - Scope of the backup policy item: days, weeks, or months.
 * `retention_value` - Value to associate with `retention_unit`.
 
+### Snapshot Distribution
```

Comment:
Can you add an example demonstrating how to use this feature on this page?

Reply:
FYI - Normally the example will be in the resource, not data source, unless it's data source specific.